test_scripts/Polices/build_options/ATF_Start_PTU_retry_sequence.lua

```diff
@@ -90,7 +96,7 @@ function Test:TestStep_Sending_PTS_to_mobile_application()
       EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "PROPRIETARY", fileType = "JSON"})
       :Do(function(_,_) time_system_request[#time_system_request + 1] = timestamp() end)
 
-      EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATING"}, {status = "UPDATE_NEEDED"}):Times(2):Timeout(64000)
+      EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"}, {status = "UPDATING"}):Times(2):Timeout(64000)
       :Do(function(exp_pu, data)
           print(exp_pu.occurences..":"..data.params.status)
           if(data.params.status == "UPDATE_NEEDED") then
```

Comment:
@istoimenova, By such change in case "UPDATE_NEEDED" is received it will be the 1st occurrence. 
In line 103 value "0" will be passed to verify_retry_sequence() function
Think "-1" has to be removed in line 103